app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt

```diff
@@ -69,19 +73,17 @@ class ErrorPanelHelper(
         errorServiceExplanationTextView.isVisible = false
         errorActionButton.isVisible = false
         errorRetryButton.isVisible = false
+        errorOpenbrowserButton.isVisible = false
     }
 
     fun showError(errorInfo: ErrorInfo) {
-
         if (errorInfo.throwable != null && errorInfo.throwable!!.isInterruptedCaused) {
             if (DEBUG) {
                 Log.w(TAG, "onError() isInterruptedCaused! = [$errorInfo.throwable]")
             }
             return
         }
 
-        ensureDefaultVisibility()
-
```

Comment:
Could you explain why you removed this?

Reply:
It's my fault, I shouldn't have deleted it